test/e2e/bootstrapper_test.go

```diff
@@ -23,7 +23,7 @@ var installDir string
khSTUJBZjhFQ0RBR0FRSC9BZ0VBTUEwRwpDU3FHU0liM0RRRUJDd1VBQTRJQkFRQmE5R0pxSHBKRzMyekdITm1DS3M4Q3VReGEwendjZHVvdnMxMkFsYWZlCmptQ21FbUx6MnVkUk1MMjByU2pEQ2Z4a1U2bTRzbnJINVdMYlpMazk0R1VjZ2lTVG1MeHMvSFZzbjJrcVhkTEMKLzRaQjcyUk0veURBZGRYaFFGRGlqbER1Yi9CT2o1OUt5UGNUNnA1SHNQMnpkRi9Yd2ViYWNVMlpUM1c5S29mZQpRcmttMFBwb2I4eERZYndTeGh3SUVxN1ZsZ3Vaalo2VEg0RTJ1dmcvT1VJZ2Q4SE9ndFhrazd4dGdPRTdVTUJqCnN2QW03VGRuWi9YLzd4YlU0cHRyK01jWk1BaXNjVk1TMXlmaUpsM3VVZ01abFNTaWNiVkF0Y0NmbDkwQU5VQXEKdzkyZWxFNHBwcTIwWjk4WlhEcXRVNWpUcTVZemlwZUMvMjhTRnNHNGdjakUKLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo%3D%0A%20%20%20%20server%3A%20https%3A%2F%2Fapi-int.ssoto-dev.devcluster.openshift.com%3A6443%0A%20%20name%3A%20local%0Acontexts%3A%0A-%20context%3A%0A%20%20%20%20cluster%3A%20local%0A%20%20%20%20user%3A%20kubelet%0A%20%20name%3A%20kubelet%0Acurrent-context%3A%20kubelet%0Apreferences%3A%20%7B%7D%0Ausers%3A%0A-%20name%3A%20kubelet%0A%20%20user%3A%0A%20%20%20%20token%3A%20eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJvcGVuc2hpZnQtbWFjaGluZS1jb25maWctb3BlcmF0b3IiLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlY3JldC5uYW1lIjoibm9kZS1ib290c3RyYXBwZXItdG9rZW4iLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3VudC5uYW1lIjoibm9kZS1ib290c3RyYXBwZXIiLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3VudC51aWQiOiI4MzIzMDk4Mi1kYTI2LTExZTktYjNhZi0wYTI4YTRlMmU2OGUiLCJzdWIiOiJzeXN0ZW06c2VydmljZWFjY291bnQ6b3BlbnNoaWZ0LW1hY2hpbmUtY29uZmlnLW9wZXJhdG9yOm5vZGUtYm9vdHN0cmFwcGVyIn0.tveE4Zf_VIqBocwKrQlX2ZWLZ0ZXxUzgF12XxI2NulvGf90M55kYxQE419kuum5J961iq02CdH0qrM_dfI6USVJ40wWfCaZG94bWEWO2fA9tXKAKBgi-9FKdrTkmDwGtCdL4KXP7kCIPnbwBiTEg7-_0P5ok1rTKSoMPP7auCaqGafZwbckgSWRe5K3UBaEk_MQeqfqIuVC0dK6c8ra5dj3WZWLYyEDxk8cAI_YBxXQ438N-G2I-Y74y_ha-pwlYcjP40Yp4ldM7GlBnGdzzD0z78aqq54KVP2HTvMx8Qte5hPi7UqsV3Xigk4YwQpxwV4I_GdalNHzTJZxRjuA6-A%0A"},"mode":420}]},"systemd":{"units":[{"contents":"[Unit]\nDescription=Kubernetes Kubelet\nWants=rpc-statd.service crio.service\nAfter=crio.service\n\n[Service]\nType=notify\nExecStartPre=/bin/mkdir --parents /etc/kubernetes/manifests\nExecStartPre=/bin/rm -f /var/lib/kubelet/cpu_manager_state\nEnvironmentFile=/etc/os-release\nEnvironmentFile=-/etc/kubernetes/kubelet-workaround\nEnvironmentFile=-/etc/kubernetes/kubelet-env\n\nExecStart=/usr/bin/hyperkube \\\n    kubelet \\\n      --config=/etc/kubernetes/kubelet.conf \\\n      --bootstrap-kubeconfig=/etc/kubernetes/kubeconfig \\\n      --kubeconfig=/var/lib/kubelet/kubeconfig \\\n      --container-runtime=remote \\\n      --container-runtime-endpoint=/var/run/crio/crio.sock \\\n      --node-labels=node-role.kubernetes.io/worker,node.openshift.io/os_id=${ID} \\\n      --minimum-container-ttl-duration=6m0s \\\n      --volume-plugin-dir=/etc/kubernetes/kubelet-plugins/volume/exec \\\n      --cloud-provider=aws \\\n       \\\n      --v=3\n\nRestart=always\nRestartSec=10\n\n[Install]\nWantedBy=multi-user.target\n","enabled":true,"name":"kubelet.service"}]}}`
```

Comment:
If possible, please keep the 2.x config as well to exercise the code path. It's fine to keep if you think this is going to be a throw away work